packages/oas-utils/src/entities/spec/requests.ts

```diff
@@ -102,7 +102,7 @@ export const oasRequestSchema = z.object({
   /** Hide operations */
   'x-internal': z.boolean().optional(),
   'x-scalar-ignore': z.boolean().optional(),
-}) satisfies ZodSchema<OpenAPIV3_1.OperationObject>
```

Comment:
this incorrectly tests the input instead of the inferred value, moved the check to a test